fotoapparat/src/main/java/io/fotoapparat/parameter/range/Ranges.java

```diff
@@ -1,11 +1,32 @@
 package io.fotoapparat.parameter.range;
 
+import android.support.annotation.NonNull;
+
+import java.util.Collection;
+
 /**
- * Class with factory methods for {@link Range}.
+ * Class with factory and utility methods for {@link Range}.
  */
 public class Ranges {
 
-    public static <T extends Comparable<T>> Range<T> range(T lowerBound, T upperBound) {
+    public static <T extends Comparable<T>> Range<T> range(@NonNull T lowerBound,
```

Comment:
JavaDocs are missing. Without them it is not entirely clear for the user of the API what is the difference between `range` and `discreteRange`.

Reply:
Maybe replace `range` with `continuousRange` and refer to `Range`'s classes JavaDocs?

Reply:
Would be even better.